autoload/lumen/platforms/macos.vim

```diff
@@ -1,22 +1,23 @@
-let s:path = resolve(expand('<sfile>:p:h')) . '/macos/watcher'
+let s:exe = resolve(expand('<sfile>:p:h')) . '/macos/watcher'
 
 func lumen#platforms#macos#watch_cmd()
-	return [s:path . ".swift"]
+	return [s:exe]
 endfunc
 
 func lumen#platforms#macos#parse_line(line)
-	if a:line == "LDark"
+	if a:line =~ "^Dark_.*"
```

Comment:
Why not use `if a:line == "Dark_"`? Using the Regex Engine is unnecessary, if we can just check for an exact match, which is more precise and even slightly faster.

Reply:
`if a:line == "Dark_"` didn't work for me. I printed the lines coming from system command and they were having a wierd character at the end, something like this: `Dark_@` or `Light@`. That's why I used regex. I could have replace it with `tr` or `sed` in the system command but vim's regex seemed easier to me.

Reply:
Just converted `system(...)` functions to `systemlist(...)[0]`. Now the `==` match works as expected. Is it okay?

Reply:
Yes, that's okay, thanks!